react/NewAddressForm.tsx

```diff
@@ -44,7 +93,28 @@ const NewAddressForm: React.FC<Props> = ({ onAddressCreated }) => {
           <LocationSearch onSelectAddress={onAddressCreated} />
         </div>
       )}
-    </Fragment>
+
+      {hasAvailableAddresses && (
+        <div className="mt6">
+          <FormattedMessage
+            id="store/checkout.shipping.backToAddressList"
+            values={{
+              // eslint-disable-next-line react/display-name
+              a: (chunks: any) => {
```

Comment:
question: on the interface, this looks like a very straight forward component, but here looks a little bit more complicated. It's supposed to be just a hyperlink, no? What is this chunk?

Reply:
it is the content inside the link, you can view the usage of `formatMessage` of `react-intl` in the third example of [this section](https://formatjs.io/docs/react-intl/api#usage) in their docs